ci/cloudbuild/publish-artifacts.yaml

```diff
@@ -1,6 +1,6 @@
 steps:
 
-- name: 'gcr.io/$PROJECT_ID/prepare-go-workspace:0.10.2'
+- name: 'gcr.io/$PROJECT_ID/prepare-go-workspace:0.11.2'
```

Comment:
Do we want the update to also include K8s changes? My understanding is the difference between v0.10 and v0.11 is the K8s version bump.